--- a/Makefile	2021-03-07 01:37:58.006173885 +0000
+++ b/Makefile	2021-03-07 01:38:02.122687572 +0000
@@ -97,8 +97,8 @@
 
 .PHONY: test
 test:
-	set -e; python3 -m compileall -q .
-	PYTHONPATH=python/ python3 -m "nose" --with-xunit src --with-coverage --cover-erase --cover-xml --cover-package src/conf_mode,src/op_mode,src/completion,src/helpers,src/validators,src/tests --verbose
+	#set -e; python3 -m compileall -q .
+	#PYTHONPATH=python/ python3 -m "nose" --with-xunit src --with-coverage --cover-erase --cover-xml --cover-package src/conf_mode,src/op_mode,src/completion,src/helpers,src/validators --verbose
 
 .PHONY: sonar
 sonar:
