From c12a6f967587868da86fe90d50248b0a1e259d5e Mon Sep 17 00:00:00 2001
From: Mathew McBride <matt@traverse.com.au>
Date: Wed, 10 Mar 2021 04:37:33 +0000
Subject: [PATCH] enable xdp build on arm64

---
 debian/control | 8 ++++----
 debian/rules   | 6 ------
 2 files changed, 4 insertions(+), 10 deletions(-)

diff --git a/debian/control b/debian/control
index 2c8ee3d4..d9a119bb 100644
--- a/debian/control
+++ b/debian/control
@@ -6,10 +6,10 @@ Build-Depends:
   debhelper (>= 9),
   fakeroot,
   gcc-multilib [amd64],
-  clang [amd64],
-  llvm [amd64],
-  libelf-dev (>= 0.2) [amd64],
-  libpcap-dev [amd64],
+  clang,
+  llvm,
+  libelf-dev (>= 0.2),
+  libpcap-dev,
   build-essential,
   libvyosconfig0 (>= 0.0.7),
   libzmq3-dev,
diff --git a/debian/rules b/debian/rules
index 8e5aee3e..c4eef825 100755
--- a/debian/rules
+++ b/debian/rules
@@ -22,10 +22,7 @@ override_dh_gencontrol:
 
 override_dh_auto_build:
 	make all
-ifeq ($(DEB_TARGET_ARCH),amd64)
-	# Only build XDP on amd64 systems
 	make vyxdp
-endif
 
 override_dh_auto_install:
 	dh_auto_install
@@ -111,9 +108,6 @@ override_dh_auto_install:
 	mkdir -p $(DIR)/$(VYOS_BIN_DIR)
 	cp -r smoketest/bin/* $(DIR)/$(VYOS_BIN_DIR)
 
-ifeq ($(DEB_TARGET_ARCH),amd64)
-	# We only install XDP on amd64 systems
 	mkdir -p $(DIR)/$(VYOS_DATA_DIR)/xdp
 	cp -r src/xdp/xdp_prog_kern.o $(DIR)/$(VYOS_DATA_DIR)/xdp
 	find src/xdp -perm /a+x -exec cp {} $(DIR)/$(VYOS_SBIN_DIR) \;
-endif
-- 
2.20.1

