From 8a3fe67c9bc471404f7721a99eddafb4e96a02b7 Mon Sep 17 00:00:00 2001
From: Mathew McBride <matt@traverse.com.au>
Date: Fri, 4 Nov 2022 22:43:42 +0000
Subject: [PATCH] vyos-1x: enable XDP on arm64

---
 debian/control | 14 +++++++-------
 debian/rules   |  4 ++--
 2 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/debian/control b/debian/control
index d7cd5b68..0b4eac39 100644
--- a/debian/control
+++ b/debian/control
@@ -7,11 +7,11 @@ Build-Depends:
   dh-python,
   fakeroot,
   gcc-multilib [amd64],
-  clang [amd64],
-  llvm [amd64],
-  libbpf-dev [amd64],
-  libelf-dev (>= 0.2) [amd64],
-  libpcap-dev [amd64],
+  clang,
+  llvm,
+  libbpf-dev,
+  libelf-dev (>= 0.2),
+  libpcap-dev,
   build-essential,
   libvyosconfig0 (>= 0.0.7),
   libzmq3-dev,
@@ -54,7 +54,7 @@ Depends:
   etherwake,
   ethtool,
   fdisk,
-  fastnetmon [amd64],
+  fastnetmon,
   file,
   frr (>= 7.5),
   frr-pythontools,
@@ -78,7 +78,7 @@ Depends:
   lcdproc,
   lcdproc-extra-drivers,
   libatomic1,
-  libbpf0 [amd64],
+  libbpf0,
   libcharon-extra-plugins (>=5.9),
   libcharon-extauth-plugins (>=5.9),
   libndp-tools,
diff --git a/debian/rules b/debian/rules
index 5a58aeeb..5caab9db 100755
--- a/debian/rules
+++ b/debian/rules
@@ -27,10 +27,10 @@ override_dh_gencontrol:
 
 override_dh_auto_build:
 	make all
-ifeq ($(DEB_TARGET_ARCH),amd64)
+#ifeq ($(DEB_TARGET_ARCH),amd64)
 	# Only build XDP on amd64 systems
 	make vyxdp
-endif
+#endif
 
 override_dh_auto_install:
 	dh_auto_install
-- 
2.30.2

